src/App.tsx

```diff
@@ -1,34 +1,61 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { useAppDispatch, useAppSelector } from './app/hooks';
+import { getTodos } from './api';
+import { actions } from './features/todos';
 
 export const App: React.FC = () => {
+  const [isLoading, setIsLoading] = React.useState(false);
+  const dispatch = useAppDispatch();
+  const currentTodo = useAppSelector(state => state.currentTodo);
+
+  const loadTodos = async () => {
+    setIsLoading(true);
+    try {
+      const todosFromServer = await getTodos();
+
+      dispatch(actions.setTodos(todosFromServer));
+    } catch {
+      Promise.reject();
```

Comment:
It's better to handle the error instead of just rejecting the promise. You can display an error message or handle it in any other appropriate way. 🚀